public/static/docs/install/index.md

```diff
@@ -7,6 +7,18 @@ Please double check that you don't already have DVC (for example running
 - [Install on Windows](/doc/install/windows)
 - [Install on Linux](/doc/install/linux)
 
+## Install as a Python library
+
+When you install DVC with an environment manager like `pip` or `conda`, the
```

Comment:
I'm being very *nitpicky* here, and it does not make a difference if you ignore this. But, `pip` is not an environment manager, it's just a package manager. However, conda is both.

In overall statement, this does not make much difference but, I just made a suggestion for correctness sake.

Reply:
No, this is great feedback! Someone else mentioned this before, I thought I had fixed it but I guess there was some repetition in those docs. Fixed in 59bb2f29 now.